components/table/table.jsx

```diff
@@ -61,6 +61,12 @@ const InfiniteTable = ({
       )
   }, [])
 
+  const bodyHasCallbacks = Boolean(sidebar)
+
+  // even if this is not an actual Hook, it starts from 'use-'
+  // to potentially have useCallback() call behind
+  const useRowCallback = (handler) => (bodyHasCallbacks ? handler : null)
```

Comment:
Why don't you add useCallback here?

Reply:
I was afraid to use it here maybe because of the boundary (which I did not check). Also, this is not needed at the moment. Currently, the function returns handler only when it's needed, it's a simple check.